src/components/TrustedTimestamping/Faq.vue

```diff
@@ -15,6 +15,7 @@ div
 export default {
   name: "TTFaq",
   props: {
+    index: { type: Number },
```

Comment:
Why do we need index? `_uid` should be unique per FAQ and we don't need to know this in the parent do we?

Reply:
_uid is deprecated so removing here.

Reply:
Looks like `_uid` is undefined? 
![image](https://user-images.githubusercontent.com/2244987/121109568-d4fb4600-c7d9-11eb-8a7b-2482414844f0.png)